src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -32,6 +32,10 @@ public class HttpClientHandlerTest
 
         private readonly NetworkCredential _credential = new NetworkCredential(Username, Password);
 
+        public readonly static object[][] GetServers = HttpTestServers.GetServers;
```

Comment:
> Xunit demands that they be public. Must have something to do with how they use reflection.

Reflecting over non-public members is a strongly discouraged practice. IIRC such members many not even be available in certain conditions.